runtime/tutor

```diff
@@ -1340,6 +1340,182 @@ actions available in a popup. This will allow you to:
 
 
 
+=================================================================
+=                  CHAPTER 13.1 CREATE NEW SPLIT                =
+=================================================================
+
+In normal mode, press ctrl-w to open the Window menu. This will
+display a pop up that reminds of most window splitting and window
+control commands available.
+
```

Comment:
- This could be simplified, ctrl-w just displays the Window menu. Is it a 'pop up', does it 'remind' us of things. I would not add all this detail.
- `a pop up that reminds of` is not grammatically correct.